index.js

```diff
@@ -42,11 +42,18 @@ function loadRcConfig(callback){
 				var options;
 
 				if(!err) {
+					if (file === "") {
+						file = "{}";
+					}				
 					try {
 						options = JSON.parse(stripJsonComments(file));
 					}
 					catch(e) {
-						err = e;
+						if (options === undefined) {
+							err = new Error("Your .jshintrc file appears to only contain comments. Please add empty curly braces to .jshintrc: {}");
+						} else {
+							err = e;
+						}
```

Comment:
Let's just throw a better error message here, no need for an `if`.

If this catch runs, 99% we failed to parse the config file, so let's just throw a similar error to what jshint cli has. Do we have access to the config file path?